srcpkgs/python3-cairocffi/template

```diff
@@ -1,10 +1,10 @@
 # Template file for 'python3-cairocffi'
 pkgname=python3-cairocffi
-version=1.2.0
-revision=2
+version=1.3.0
+revision=1
 wrksrc="cairocffi-${version}"
 build_style=python3-module
-hostmakedepends="python3-setuptools python3-cffi python3-wheel"
+hostmakedepends="python3-setuptools python3-cffi python3-wheel python3-xcffib"
```

Comment:
Is `python3-xcffib` a mandantory dependencies, could this package be built without it?

Haskell isn't available in ppc

Reply:
Yes I belive it's mandatory from https://docs.qtile.org/en/stable/manual/install/index.html#core-dependencies
`Qtile uses cairocffi for drawing on status bars and popup windows. Under X11, cairocffi requires XCB support via xcffib, which you should be sure to have installed before installing cairocffi, otherwise the needed cairo-xcb bindings will not be built.`

Reply:
```sh
hostmakedepends="python3-setuptools python3-cffi python3-wheel $(vopt_if xcb python3-xcffib)"
build_options="xcb"
case "$XBPS_MACHINE" in
x86_64*|i686|ppc64le*|ppc64)
    build_options_default="xcb"
esac
```